github.js

```diff
@@ -184,9 +186,10 @@ function downloadRepos(repos, callback) {
             }
 
             Logger.log("Repository: " + c.full_name, "progress");
-            repo.exec("clone " + c.ssh_url + " " + path, function (err) {
+            repo.exec("clone " + c.clone_url + " " + path, function (err) {
```

Comment:
Ideally, this should be configurable. But https will not work with 2FA for private repositories. I would suggest keeping the ssh_url for now or making it configurable.

Reply:
Ok, kept the ssh link as is.